api-migration.md

```diff
@@ -40,8 +40,31 @@ The final response isn't returned to the client until all the plugin hooks have
 been executed. This means that any pipe on the `cloudant.request` object must be
 in place _before_ the `response` event is triggered (as shown above).
 
+If you were using the 429 `retry` plugin in version 1.x then be aware that
```

Comment:
Since this is a breaking change, should we create a sub-section header here and possibly have a link in the CHANGES log (maybe README)?

Reply:
good thinking, will add.